src/plugins/dashboard/public/application/dashboard_app.tsx

```diff
@@ -43,13 +42,12 @@ export function DashboardApp({
   const { onAppLeave } = useDashboardMountContext();
   const {
     chrome: { setBreadcrumbs, setIsVisible },
+    screenshotMode: { isScreenshotMode },
     coreContext: { executionContext },
-    data: { search },
     embeddable: { getStateTransfer },
     notifications: { toasts },
-    screenshotMode: { isScreenshotMode },
     settings: { uiSettings },
-    spaces: { getLegacyUrlConflict },
+    data: { search },
```

Comment:
I always organize imports and de-structuring by number of characters to make the file look pretty ✨. The way I do it is definitely not something we should standardize in my opinion hahah. So that leaves two options here:

**Option 1**: We make alphabetical imports / de-structuring our team's **standard**, and we look into making a prettier / eslint rule to automatically do that for us.

**Option 2**: We leave it totally up to the engineer who is editing the file.

I'm totally okay with either option! @cqliu1, this is something the 3 of us should probably think about.

Reply:
I'm also good with either! Definitely something worth discussing so we can all get on the same page, since I know it's come up a few times at this point :+1: